assets/js/helpers/boot.js

```diff
@@ -76,3 +78,16 @@ $(document).ready(function() {
   });
   window.bootTopic();
 });
+
+getScript = function(src, callback) {
```

Comment:
didn't change this actually, must have checked out some other version. I'll delete it